include/gar/graph_info.h

```diff
@@ -873,6 +891,7 @@ class GraphInfo {
 
  private:
   std::string name_;
+  InfoVersion version_;
```

Comment:
Is the "GetVersion" function required for VertexInfo, EdgeInfo and GraphInfo?

Reply:
Yes, I think `GetVersion` can be add as a method for user to access the version